tools/fiptool/fiptool.c

```diff
@@ -629,10 +662,11 @@ static void create_usage(void)
 
 static int update_cmd(int argc, char *argv[])
 {
-	struct option opts[toc_entries_len + 2];
+	struct option opts[toc_entries_len + 3];
 	char outfile[FILENAME_MAX] = { 0 };
 	fip_toc_header_t toc_header = { 0 };
 	unsigned long long toc_flags = 0;
+	unsigned long align = 1;
 	int pflag = 0;
 	int i;
 
```

Comment:
add_opt(opts, i, "align", required_argument, OPT_ALIGN);

This is missing in the update and remove subcommands.  I have fixed this in the rebase that I did.